src/multivariates.jl

```diff
@@ -230,15 +230,13 @@ end
 function logpdf(d::MultivariateDistribution, X::AbstractMatrix)
     size(X, 1) == length(d) ||
         throw(DimensionMismatch("Inconsistent array dimensions."))
-    T = promote_type(partype(d), eltype(X))
-    _logpdf!(Vector{T}(undef, size(X,2)), d, X)
+    map(i -> _logpdf(d, view(X, :, i)), axes(X, 2))
```

Comment:
As mentioned in the original PR,
```suggestion
    map(Base.Fix1(_logpdf, d), eachcol(X))
```
would requires Julia >= 1.1 or Compat.